### ISSUE TITLE
data2vec audio pretrained model

### ISSUE THREAD

**Issue by maziye-azad** [facebookresearch/fairseq#4165]
Hi, sorry to ask such a trivial question, I am new to fairseq and i dont know how i can get data2vec audio pretrained model in colab.
any help is appreciated.

**Comment by Yingfang-Yang** [comment]
you can try like: wget  https://dl.fbaipublicfiles.com/fairseq/data2vec/audio_base_ls.pt

**Comment by maziye-azad** [comment]
> you can try like: wget https://dl.fbaipublicfiles.com/fairseq/data2vec/audio_base_ls.pt

that's right,thank you! but how can i get the architecture of the model? should i write it myself?
